README.md

```diff
@@ -65,7 +66,7 @@ You can copy values of other environment variables by customizing
 
 This function may also be called interactively.
 
-Note that your shell will inherit Emacssenvironment variables when
+Note that your shell will inherit Emacs's environment variables when
```

Comment:
All the documentation I could find (Emacs man page, info, online docs) all appear to use "Emacs's", so I went with that. I can still change it if you'd prefer.